primitives/inherents/src/lib.rs

```diff
@@ -398,9 +398,9 @@ impl<E: codec::Encode> IsFatalError for MakeFatalError<E> {
 	}
 }
 
-/// A module that provides an inherent and may also verifies it.
+/// A pallet that provides an inherent and may, optionally, verify it.
```

Comment:
it is actually also optional to provide an inherent, this trait can be used just to verify an inherent